### ISSUE TITLE
idx.register function with activationToken returns Bad Request error ("access_denied" and "The requested feature is not enabled in this environment." error.).

### ISSUE THREAD

**Issue by sb-developers** [okta/okta-auth-js#1297]
### Describe the bug?

When we call to idx.register({activationToken: this.token, authenticator: 'okta_password',  password: this.newPasswordForm.value.password}) function returns "Bad Request". Concretically, returns "access_denied" and "The requested feature is not enabled in this environment." error.

### What is expected to happen?

We expect to register user with his new Password and activate user.

### What is the actual behavior?

Register user with his new Password and activate user.

### Reproduction Steps?

In this moment we don't have access online: 
1) We create register page only getting email. When submit register, we call to Okta’s "users" (POST) API.
2) This first step returns User Activation email where we’ve got a button to redirect a new page to confirm email and add his password. This URL contains activation_token to register.
3) When submit password, we call to this function: idx.register({activationToken: this.token, authenticator: ‘okta_password’, password: this.newPasswordForm.value.password}).
4) Returns error

### SDK Versions

@okta/okta-angular: ^5.1.1
@okta/okta-auth-js: ^6.1.0
@okta/okta-signin-widget: ^6.0.1

### Execution Environment

Browser

### Additional Information?

Angular application

**Comment by oleksandrpravosudko-okta** [comment]
Thanks for report @sb-developers.
Can you please confirm your org's profile enrollment policy is configured to allow sign up?

**Comment by sb-developers** [comment]
Hi Oleksandr,
Thanks for your response.
I'm trying to search this page into our Okta Admin menu, but we don't find nothing like this. Where will be appear this page?
If we didn0't found it, What could we do?
Thanks,

**Comment by denysoblohin-okta** [comment]
"Security -> Profile Enrollment" - list of Profile Enrollment policies including "Default Policy".
Click on Edit icon to open edit page.
Under "Enrollment settings" select "Actions -> Edit".
In "Edit rule" pop-up find "For new users: Sign-up - Allowed / Denied"

Also you can reach out to support@okta.com

<img width="1056" alt="Screenshot 2022-09-12 at 11 56 35" src="https://user-images.githubusercontent.com/72614880/189613834-a3394c94-6637-47a7-b20a-8a9dcd5e7b2b.png">

**Comment by sb-developers** [comment]
Hi Oleksandr,
Thanks for your response.
No... If you can see in the image, we don't have this section in our Okta Admin menu. What can we do?

![image](https://user-images.githubusercontent.com/53536734/189620014-899fce83-d33f-47ff-98b6-4eefe8e6320a.png)

**Comment by denysoblohin-okta** [comment]
Please reach out to support@okta.com . Self-Service Registration feature should be enabled for your account

**Comment by sb-developers** [comment]
Ok, we will talk with Okta support. 
Thanks!

**Comment by denysoblohin-okta** [comment]
@sb-developers Have you found a solution? If yes, can we close this issue?

**Comment by ikermonk** [comment]
No, we didn't. Okta support didn't answer us.... But you can close this issue.
